src/rt/test/harness.h

```diff
@@ -135,6 +135,8 @@ class SystematicTestHarness
                 << duration_cast<seconds>((t1 - start)).count() << " seconds"
                 << std::endl;
     }
+
+    std::cout << "Finished!" << std::endl;
```

Comment:
What finished? Here we know, but in the middle of a log, it may be harder to pinpoint.

Specific messages like "Execution of test <test> finished" makes it easier, as well as to find in code when looking for a break.